observability/tools/loki.md

```diff
@@ -0,0 +1,39 @@
+# Loki
+
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system, created by Grafana
+Labs inspired by the learnings from Prometheus. Loki is commonly referred as Prometheus but for logs, which
+makes total sense once both tools follow the same architecture, which is a agent collecting metrics in each
+of the components of the software system, a server which stores the logs and also the Grafana dashboard, which
+access the loki server to build its visualizations and queries. That being said, Loki has three main
+components:
+
+## Promtail
+
+Its the agent portion of Loki. It can be used to grab metrics from several different places, like var/log/ for
+example. The configuration of the Promtail is a yaml file called ```config-promtail.yml``` commonly store on
+the  ```/usr/local/bin``` path. In this file, its described all the paths and log sources that will be
+aggregated on Loki Server.
+
+## Loki Server
+
+Responsible for receiving and storing all the logs received from all the different systems. The Loki is also
+responsible for the queries done on Grafana, for example.
```

Comment:
The queries from Grafana are processed on Loki Server?

Reply:
Yes! At least is what the doc says!

Reply:
Yes, they are. I've contributed code to Loki in the past and understand its arch quite well.